dds_registration/templates/dds_registration/profile.html.django

```diff
@@ -52,6 +52,14 @@
   <p>Not yet a DdS member. <a href="{% url 'membership_application' %}">Apply for membership</a>.</p>
 {% endif %}
 
+{% if user.membership.payment.status == 'ISSUED' %}
```

Comment:
Might need to check here if there is a payment - honorary memberships won't be linked to a `Payment` object at all.